interface/modules/zend_modules/public/index.php

```diff
@@ -39,6 +39,8 @@
 require_once(dirname(__FILE__)."/../../../../library/forms.inc");
 require_once(dirname(__FILE__)."/../../../../library/options.inc.php");
 
+require_once(dirname(__FILE__)."/../../../../library/sql_upgrade_fx.php");
```

Comment:
This file no longer exists, so line can be removed

Reply:
oops, disregard above post. I was confusing it with the acl upgrade library.

Reply:
It is odd, however, to bring this in here. Can't it be called in the scripts that use these functions?

Reply:
Without using "library / sql_upgrade_fx.php" the program throws an error: "Call to undefined function Installer \ Controller \ upgradeFromSqlFile ()"